src/main/java/org/opentripplanner/middleware/controllers/api/OtpUserController.java

```diff
@@ -100,6 +103,14 @@ protected void buildEndpoint(ApiEndpoint baseEndpoint) {
                     .withResponseType(OtpUser.class),
                 TrustedCompanion::acceptDependent
             )
+            .get(path(ROOT_ROUTE + String.format(GET_DEPENDENT_MOBILITY_PROFILE_PATH, ID_PARAM))
+                    .withDescription("Retrieve a dependent's mobility profile.")
+                    .withResponses(SwaggerUtils.createStandardResponses(MobilityProfile.class))
+                    .withPathParam().withName(ID_PARAM).withRequired(true).withDescription("The id of the related user.").and()
```

Comment:
I think the caller only has access to emails and does not have access to the user Mongo IDs... should they?

Reply:
If you prefer using the id over the email as parameter for this endpoint, I think we could add a userId field to `RelatedUser` and populate that on confirmation/deletion from the companion user.

Reply:
Disregard, we are using a list of dependents which is stored as a list of ids.